FlaskApi/src/auth.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask
+from flask_restful import Resource, Api, reqparse
+
+app = Flask(__name__)
+api = Api(app, prefix='/api/v1/auth')
+
+
+signup = {
+    "users1": {"name":"Anthony Muna","email":"anthony@gmail.com", "password":"12345"},
+    "users2": {"name":"Carol Kamau", "email":"carol@gmail.com", "password":"54321"},
+}
+
+login = {
+    "user1": {"name": "Anthony Muna", "email":"anthony@gmail.com"},
+    "user2": {"name":"Carol Kamau", "email":"carol@gmail.com"},
+}
+
+parser = reqparse.RequestParser()
+
+class signin(Resource):
+    def get(self):
+        return login
```

Comment:
This will return all information in the dictionary `login` instead of information necessary after signing in a user.